.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!-- 
+Thanks for contributing a pull request to SimPEG user tutorials!
+Remember to use a personal fork of SimPEG user tutorials to propose changes.
+
+Check out the stages of a pull request at
+https://docs.simpeg.xyz/content/getting_started/contributing/pull-requests.html
+
+Note that we are a team of volunteers and we appreciate your
+patience during the review process.
+
+Again, thanks for contributing!
+
+Feel free to remove lines from this template that do not apply to you pull request.
+-->
+
+#### Summary
+<!-- Add a summary of this Pull Request. Explain what it provides as tutorial material. -->
+
+#### PR Checklist
+* [ ] If this is a work in progress PR, set as a Draft PR
+* [ ] Introduction is complete:
+    * [ ] Title and author added to notebook
+    * [ ] Admonitions for notebook difficulty and computational resources have been added
+    * [ ] Keywords list has been added
+    * [ ] Summary paragraph describing the tutorial has been added
+    * [ ] Learning objectives have been listed
+    * [ ] Hyperlinks to other tutorial notebooks added if necessary
+* [ ] For sections and subsections:
+    * [ ] A header and short summary of what is being done
+    * [ ] Links to API documentation added for all classes and functions that are used
+    * [ ] Newly introduced functionality is explained or links provided to relevant notebooks
+    * [ ] All figures are legible and rendered appropriately
+    * [ ] Coding cells have been linted according to the [style guides](https://docs.simpeg.xyz/latest/content/getting_started/contributing/code-style.html).
```

Comment:
We have style guide for this particular repo. I'd suggest linking at those instead of linking to the ones we have in SimPEG.
```suggestion
    * [ ] Coding cells have been linted according to the [style guides](https://simpeg.xyz/user-tutorials/formatting#check-style-of-notebooks).
```